migrations/20240722180526_drop_16_model_goal_v2/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE `_Goal_user` (
+    `A` VARCHAR(191) NOT NULL,
+    `B` VARCHAR(191) NOT NULL,
+
+    UNIQUE INDEX `_Goal_user_AB_unique`(`A`, `B`),
+    INDEX `_Goal_user_B_index`(`B`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- AddForeignKey
+ALTER TABLE `_Goal_user` ADD CONSTRAINT `_Goal_user_A_fkey` FOREIGN KEY (`A`) REFERENCES `Goal`(`id`) ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE `_Goal_user` ADD CONSTRAINT `_Goal_user_B_fkey` FOREIGN KEY (`B`) REFERENCES `User`(`id`) ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
I believe having 2 migration files is a no-no for this setup. But I'm not sure of the right protocol to delete one without getting errors. If anyone has any feedback on this I'd appreciate it!

Reply:
After speaking to Josh, having 2 migration files is okay in this case because we are not yet pushing to the production db. Still hoping to get a couple Peer Reviews on this. Thanks!